### PR TITLE
readme: some updates to the README

### PR DESCRIPTION
**Pull request by obnoxxx**
improve some links and mention that the containers are used by ceph.

### CODE REVIEW

**Review by synarete — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -3,12 +3,13 @@
 Container images for [Samba](https://samba.org) services.
 
 ### Our images:
-* Are [OCI](https://opencontainers.org/) formatted container images
-* Provide application-like high-level entrypoint scripts
+* Are [OCI](https://opencontainers.org/) formatted container images.
+* Provide application-like high-level entrypoint scripts.
```
Comment (phlogistonjohn) [inline README.md:7]: These bullet points are sentence fragments. They do not need punctuation. Adding punctuation to only some items and not all of them makes the content inconsistent. Please remove the periods that this PR is adding

**Review by phlogistonjohn — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (obnoxxx) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ Container images for [Samba](https://samba.org) services.
 * Are used by the [samba-operator](https://github.com/samba-in-kubernetes/samba-operator) for Kubernetes
 * Don't require Kubernetes
 * Are [available at quay.io](https://quay.io/organization/samba.org)
+* Are used by the [ceph project](https://github.com/ceph/ceph) to provide  SMB services on CephFS.
```
Comment (phlogistonjohn) [inline README.md:12]: this one still has a trailing period at the end of the line

**Review by phlogistonjohn — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (obnoxxx) [reply README.md]: done. removed

**Review by phlogistonjohn — APPROVED** [review]

**Review by synarete — APPROVED** [review]
lgtm

**Pull request merged**